src/System.Device.Gpio/System/Device/Gpio/GpioDriver.cs

```diff
@@ -14,6 +14,14 @@ public abstract class GpioDriver : IDisposable
         /// </summary>
         protected internal abstract int PinCount { get; }
 
+		/// <summary>
+        /// Set the GPIO bank to be used /dev/gpiochip0 ... /dev/gpiochipX
+        /// </summary>
+		protected internal virtual void SetDefaultGpioChip(int devGpioChip) 
```

Comment:
is it virtual for back compat? I'd go with abstract since we haven't shipped yet.

Reply:
i think we should set the chip number on driver i guess we don't want to update all drivers with this method, preferably set it with driver constructor argument.